package.json

```diff
@@ -25,7 +25,15 @@
     "url": "https://github.com/blackbaud/skyux-sample/issues"
```

Comment:
The keyword "sample" is still being used in the package.json, skyuxconfig.json, and README files.

Reply:
done